src/views/endorse-company-closure-certificate.njk

```diff
@@ -0,0 +1,74 @@
+{% extends 'layout.njk' %}
+
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+
+{% block pageTitle %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% include 'components/errors/error-summary.njk' %}
+
+      <h1 class="govuk-heading-xl">Sign the application</h1>
+      <h2 class="govuk-heading-m">Apply to strike off and dissolve a company</h2>
+      <h2 class="govuk-heading-m govuk-!-margin-bottom-9">
+        <span id="companyName">Company name: {{ approvalModel.companyName }}</span>
+        <span id="companyNumber">Company number: {{ approvalModel.companyNumber }}</span>
```

Comment:
Surprised this works, but if it works it works

Reply:
Added divs